README.md

```diff
@@ -17,6 +17,12 @@ A Clojure java-client wrapper for Couchbase Server 4.
     
     ;; we create a bucket
     (def bucket (c/open-bucket cluster "gamesim-sample"))
+
+    ;; we authenticate with the bucket password, note: the password parameter on the c/bucket-open
```

Comment:
That comment will help to have it in the c/authenticate method too

Reply:
Done.